Documentation/kubernetes-on-aws-render.md

```diff
@@ -198,6 +198,21 @@ hostedZone: staging.example.com
 
 If `createRecordSet` is not set to true, the deployer will be responsible for making externalDNSName routable to the controller IP after the cluster is created.
 
+### Multi-AZ Clusters
+
+Kube-aws supports "spreading" a cluster across any number of Availability Zones in a given region.
+
+```yaml
+ subnets:
+   - availabilityZone: us-west-1a
+     instanceCIDR: "10.0.0.0/24"
+   - availabilityZone: us-west-1b
+     instanceCIDR: "10.0.1.0/24"
+```
+__A word of caution about EBS and Persistent Volumes__: EBS volumes are constrained to a single Availability Zone- which means that in a Multi-AZ cluster pod specs cannot directly specify which EBS to mount via a persistent volume. Any pod spec which is being deployed in a Multi-AZ cluster must mount in EBS-backed persistent volumes via [Persistent Volume Claims](http://kubernetes.io/docs/user-guide/persistent-volumes/#persistentvolumeclaims).
```

Comment:
I've read this a few times and can't really parse it. It seems like you're telling me to not use a persistent volume, but then I see that same phrase in "persistent volume claim", which is the thing that I _should_ use?

Can this be simplified by just telling you what to do, instead of what not to do?